.changeset/honest-candles-lick.md

```diff
@@ -0,0 +1,6 @@
+---
+"dom-accessibility-api": patch
+---
+
+Previously, the accessible name of `<th>` is columnheader regardless to the scope value.
+It now treat as `rowheader` or `columnheader` by the scope value of `<th>`.
```

Comment:
You probably meant "role" not "accessible name"? Note that this won't fix https://github.com/testing-library/dom-testing-library/issues/930

Reply:
Oh, I was wrong about this. I fixed to role.

> Note that this won't fix testing-library/dom-testing-library#930

Do you mean because need to modify this library and import it into `testing-library`? Or that the modifying place are different?

Reply:
Maybe it is used `computeAccessibleName` method in `dom-testing-library`. Probably I should fix this method. 
https://github.com/testing-library/dom-testing-library/search?q=dom-accessibility-api